### ISSUE TITLE
Checkbox styling with alpha bullets

### ISSUE THREAD

**Issue by pondersson** [minad/org-modern#138]
It seems like checkboxes currently aren't stylized in alpha bullet lists. I found that changing the relevant regexp in [org-modern--make-font-lock-keywords](https://github.com/minad/org-modern/blob/main/org-modern.el#L662) from `[0-9]` to `[0-9a-zA-Z]` fixed the issue, but I'm not sure if that has any unwanted side-effects.

**Comment by minad** [comment]
Thanks for the report. As far as I understand, checkboxes are not allowed behind alpha bullets, since Org doesn't apply the org-checkbox face there.

**Comment by pondersson** [comment]
You're right. Not sure if it's intentional or just an oversight though. Can't find any mention of it in either code or documentation, and looking at the git history it seems like the current regexp is from 2010 whereas alphabetical bullet support was added in 2011. I'll raise it upstream.

**Comment by pondersson** [comment]
Turns out it was a bug and it's been fixed: https://git.savannah.gnu.org/cgit/emacs/org-mode.git/commit/?id=a07260856

**Comment by minad** [comment]
Very good. Can we reuse the regexp?
